src/qc.rs

```diff
@@ -67,7 +64,7 @@ pub trait QuorumCertificate<A: AggregateableSignatureSchemes + Serialize + for<'
         qc_vp: &Self::QCVerifierParams,
         message: &GenericArray<A::MessageUnit, Self::MessageLength>,
         qc: &Self::QC,
-    ) -> Result<Self::CheckedType, PrimitivesError>;
+    ) -> Result<(), PrimitivesError>;
```

Comment:
I think I remember why this is here now. It's not the `Checked<T>`. And it is needed.

I'm even the person who requested it, I just forgot.

And I think the entire discussion motivating this was on github.

... 90 minutes of searching later...

https://github.com/EspressoSystems/hotshot-primitives/pull/27/files/907f07d85af77fee0ba1b328a77944cdce4e64d5#r1195601782

Short version: we absolutely, non-negotiably, *need* the `CheckedType` if we want to be able to use this from HotShot.

Reply:
thanks for digging this comment for us! could you clarify what would the number `CheckedType = U256` signify?
the number of votes inside the qc?

Reply:
In the case of HotShot, the total number of voted shares, I think...

Reply:
added it back, see if you like [c9429f0](https://github.com/EspressoSystems/hotshot-primitives/pull/64/commits/c9429f0f5bb4d063bd1b5f2f3d87a24bd3879e02) @nyospe